src/environment/ale_state.cpp

```diff
@@ -12,11 +12,16 @@
 #include "ale_state.hpp"
 #include "../emucore/m6502/src/System.hxx"
 #include "../emucore/Event.hxx"
+#include "../emucore/Deserializer.hxx"
+#include "../emucore/Serializer.hxx"
 #include "../common/Constants.h"
-using namespace std;
+#include "../games/RomSettings.hpp"
 
+#include <sstream>
 #include <stdexcept>
 
+using namespace std;
```

Comment:
We should get rid of the using namespace std.

Reply:
Fixed.